tracer/test/Datadog.Trace.Tools.Runner.IntegrationTests/Checks/ProcessBasicChecksTests.cs

```diff
@@ -193,6 +200,116 @@ public async Task Working()
                 CorProfilerPath64Key);
         }
 
+        [SkippableFact]
+        public async Task WorkingWithContinuousProfiler()
```

Comment:
Just to confirm, did you skip enabling these for Windows because you didn't want to mess with CI? 😉

Reply:
:awkward-monkey: